libindy/src/api/mod.rs

```diff
@@ -89,6 +89,27 @@ pub enum ErrorCode
     // Attempt to open encrypted wallet with invalid credentials
     WalletAccessFailed = 207,
 
+    // Input provided to wallet operations is considered not valid
+    WalletInputError = 208,
+
+    // Decoding of wallet data during input/output failed
+    WalletDecodingError = 209,
+
+    // Storage error occurred during wallet operation
+    WalletStorageError = 210,
+
+    // Error during encryption-related operations
+    WalletEncryptonError = 211,
+
+    // Requested wallet item not found
+    WalletItemNotFound = 212,
+
+    // Returned if wallet's add_record operation is used with record name that already exists
+    WalletItemAlreadyExists = 213,
```

Comment:
What is the difference with existing WalletAlreadyExistsError ?

Reply:
WalletAlreadyExists - when wallet itself exists already (wallet with given name)
WalletItemAlreadyExists - when the wallet item already exists (wallet record)